fusionfrontend/src/app/fleet/components/content/asset-series-create/asset-series-create-general-information/asset-series-create-general-information.component.html

```diff
@@ -13,46 +13,45 @@
 under the License.
 -->
 
-<div class="asset-series-create-step-two">
-  <div class="content">
-      <div id="colum left">
-        <div class="title">General Information</div>
-
-        <div *ngIf="assetSeries">
-            <div>
-              <label class="if-label">Select Asset Type Template</label>
-              <p-dropdown
-                [options]="assetTypeTemplates$ | async"
-                [optionLabel]="'name'"
-                [optionValue]="'id'"
-                name="assetTypeTemplate"
-                placeholder="Select Asset Type Template..."
-                (ngModelChange)="updateTypeTemplate.emit($event)"
-                [(ngModel)]="assetSeries.assetTypeTemplateId"
-                [disabled]="mode != ViewMode.CREATE">
-              </p-dropdown>
-          </div>
-
-          <div class="clr-form-control">
-            <label class="if-label">Name the Asset or Series you want to implement</label>
-            <input [disabled]="!assetSeries" [(ngModel)]="assetSeries.name"
-                   placeholder="Asset series name">
-          </div>
+<div class="asset-series-create-general-information">
+  <form [formGroup]="assetSeriesForm">
+    <div class="content with-rows">
+      <div id="column-left">
+        <div class="wizard-title">General Information</div>
 
+        <!--<div *ngIf="assetSeries"> // TODO tse: why was this ngIf in here? -->
```

Comment:
to prevent errors the ngif wait with DOM loading while the assetseries is loaded